File: frontend/smrpo-app/src/app/models/UserStory.ts
import { UserStoryPriority } from '../enums/UserStoryPriority';
import { UserStoryStatus } from '../enums/UserStoryStatus';
import { Sprint } from './Sprint';

export class UserStory {
    id: string;
    name: string;
    description: string;
    priority: UserStoryPriority;
    businessValue: number;
    comment: string;
    acceptanceTests: string[];
    status: UserStoryStatus;
    sprint: Sprint;
}

File: backend/smrpo-be/Data/Models/UserStory.cs
using smrpo_be.Data.Enums;
using System.Collections.Generic;
using System;

namespace smrpo_be.Data.Models
{
    public class UserStory : Entity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public UserStoryPriority Priority { get; set; }
        public int BusinessValue { get; set; }
		public Guid ProjectId { get; set; }
        public Guid? SprintId { get; set; }
        public UserStoryStatus Status { get; set; }
        public string Comment { get; set; }
        public bool Deleted { get; set; }

        //Relations
        public virtual Project Project { get; set; }
        public virtual IEnumerable<AcceptanceTest> AcceptanceTests { get; set; }
        public virtual IEnumerable<UserStoryTime> UserStoryTimes { get; set; }
        public virtual Sprint Sprint { get; set; }
        public virtual IEnumerable<UserStoryTask> Tasks { get; set; }
    }
}
